<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HP" name="org.test2.Test2.Datadrivrn" tests="1" failures="0" timestamp="29 Jul 2021 06:35:55 GMT" time="0.042" errors="1">
  <testcase name="tco" time="0.042" classname="org.test2.Test2.Datadrivrn">
    <error type="org.testng.TestNGException" message="
The data provider is trying to pass 2 parameters but the method org.test2.Test2.Datadrivrn#tco takes 1">
      <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 2 parameters but the method org.test2.Test2.Datadrivrn#tco takes 1
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1229)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1122)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </error>
  </testcase> <!-- tco -->
</testsuite> <!-- org.test2.Test2.Datadrivrn -->
